crates/biome_js_analyze/src/lint/correctness/use_jsx_key_in_iterable.rs

```diff
@@ -36,6 +38,27 @@ declare_lint_rule! {
     /// data.map((x) => <Hello key={x.id}>{x}</Hello>);
     /// ```
     ///
+    /// ## Options
+    ///
+    /// ```json,options
+    /// {
+    ///     "options": {
+    ///         "checkShorthandFragments": true
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// ### checkShorthandFragments
+    ///
+    /// React fragments can not only be created with `<React.Fragment>`, but also with shorthand
+    /// fragments (`<></>`). To also check if those require a key, pass `true` to this option.
+    ///
+    /// #### Usage example
+    ///
+    /// ```jsx,expect_diagnostic,use_options
+    /// data.map((x) => <>{x}</>);
+    /// ```
+    ///
```

Comment:
```suggestion
    /// ## Options
    ///
    /// ### checkShorthandFragments
    ///
    /// React fragments can not only be created with `<React.Fragment>`, but also with shorthand
    /// fragments (`<></>`). To also check if those require a key, pass `true` to this option.
    ///
    /// ```json,options
    /// {
    ///     "options": {
    ///         "checkShorthandFragments": true
    ///     }
    /// }
    /// ```    
    /// ```jsx,expect_diagnostic,use_options
    /// data.map((x) => <>{x}</>);
    /// ```
    ///
```